x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/hooks/use_styles.ts

```diff
@@ -45,17 +45,18 @@ export const useStyles = () => {
     }
     & .euiDataGridRowCell {
       font-size: ${euiTheme.size.m};
-    }
-    & .euiDataGridRowCell__actions > [data-test-subj='euiDataGridCellExpandButton'] {
-      display: none;
-    }
-    & .euiDataGridRowCell__actions--overlay {
-      padding: 0;
-    }
-
-    & .euiDataGridRowCell__contentWrapper {
+      // Vertically center content
       display: flex;
       align-items: center;
+
+      // Stretch content to full-width
+      .euiDataGridRowCell__content {
+        flex-grow: 1;
+      }
+    }
+    /* EUI QUESTION: Why is this being done via CSS instead of setting isExpandable: false in the columns API? */
+    & .euiDataGridRowCell__actions > .euiDataGridRowCell__expandCell {
+      display: none;
     }
```

Comment:
@elastic/kibana-cloud-security-posture this CSS override concerns me a little because it could easily look broken with the new cell actions design. If the cells in this data grid should not be expandable, I would like to switch to setting that via `isExpandable` API and removing this `display: none`.